src/test/java/com/google/javascript/clutz/type_renaming_with_emit_platform_externs.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Test that during the conversion of platform externs, the types below
+ * aren't renamed
+ * @externs
```

Comment:
this is confusion, the file is not an extern, but you are adding @externs annotation.